### PR TITLE
add DataRange with filters,fix filters,add statisticWallet

### PR DESCRIPTION
**Pull request by okrdima**
add DataRange with filters,fix filters,add statisticWallet

### CODE REVIEW

File: src/app/domains/Statistic/components/CollapseDateRange/CollapseDateRange.template.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react'
+import PropTypes from 'prop-types'
+import { useStyles } from './CollapseDateRange.styles'
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemText from '@material-ui/core/ListItemText'
+import Collapse from '@material-ui/core/Collapse'
```
Comment (olenashevchuk) [inline src/app/domains/Statistic/components/CollapseDateRange/CollapseDateRange.template.js:7]: use destructuring here

File: src/app/domains/Statistic/components/CollapseWallet/CollapseWallet.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import Collapse from '@material-ui/core/Collapse'
```
Comment (olenashevchuk) [inline src/app/domains/Statistic/components/CollapseWallet/CollapseWallet.js:3]: destructuring here and below

File: src/app/domains/Statistic/components/CollapseDateRange/CollapseDateRange.template.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import ExpandLess from '@material-ui/icons/ExpandLess'
+import ExpandMore from '@material-ui/icons/ExpandMore'
```
Comment (sashka2131) [inline src/app/domains/Statistic/components/CollapseDateRange/CollapseDateRange.template.js:9]: use destructuring here for icons import

File: src/app/domains/Statistic/components/CollapseWallet/CollapseWallet.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react'
```
Comment (sashka2131) [inline src/app/domains/Statistic/components/CollapseWallet/CollapseWallet.js:1]: Why file named without marks such as template or stories?

File: src/app/domains/Statistic/components/FiltersWithCollapse/FiltersWithCollapse.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types'
```
Comment (sashka2131) [inline src/app/domains/Statistic/components/FiltersWithCollapse/FiltersWithCollapse.js:1]: And this file, rename file with appropriate marks (.template.js or other)

**Pull request merged**